docs/source/format/specification.rst

```diff
@@ -36,6 +36,30 @@ provides a place to hold ownership of the in-memory database.
 - Go: ``Driver``
 - Java: ``org.apache.arrow.adbc.core.AdbcDatabase``
 
+URIs
+----
+
+When specifying the URI of the database, there is a standard option
+name to pass to drivers. This is the string `uri`, available in
+the implementations via helpers:
+
+- C/C++: :c:macro:`ADBC_OPTION_URI`
+- Go: ``OptionKeyURI``
+- Java: ``org.apache.arrow.adbc.core.AdbcDriver#PARAM_URL``
```

Comment:
I guess we should use URI, and `@Deprecated` the URL one

Reply:
sure thing